src/client/src/assets/i-reset.svg

```diff
@@ -1,3 +1,3 @@
-<svg alt="reset all pages" width="12" height="13" viewBox="0 0 12 13" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg alt="reset all pages" width="12" height="13" viewBox="0 0 12 13" fill="none" xmlns="http://www.w3.org/2000/svg"><title>Reset pages</title>
```

Comment:
This one doesn't have a role is this intended?

Reply:
Oops, I accidentally missed that but I just added the role in :)